### PR TITLE
Add K8ssandra Blueprint and example

### PR DESCRIPTION
**Pull request by PrasadG193**
Signed-off-by: Prasad Ghangal <prasad.ghangal@gmail.com>

## Change Overview

- Add a Blueprint for backup/restore K8ssandra app
- Add an example README for K8ssandra backup/restore steps

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [x] :world_map: Documentation
- [ ] :robot: Test

## Test Plan
- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Comment by PrasadG193** [comment]
Thanks for the review @ihcsim, will address them shortly

File: examples/k8ssandra/README.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+## Installing the K8ssandra
+
+> Complete guide to provision K8s cluster locally with required resources and installing K8ssandra can be found [here](https://docs-v1.k8ssandra.io/install/local/).
+
+Before we begin, make sure that there exists a storageclass with **VOLUMEBINDINGMODE of WaitForFirstConsumer.**
+
+
+### Create S3 creds secret for Medusa Amazon S3 integration
+
+K8ssandra Operator deploys Medusa to support backup and restore operations of Apache Cassandra® tables.
```
[the remark this answers is not in the document]
Comment (PrasadG193) [reply examples/k8ssandra/README.md:28]: not required

File: examples/k8ssandra/README.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+metadata:
+ name: medusa-bucket-key
+type: Opaque
+stringData:
+ # Note that this currently has to be set to medusa_s3_credentials!
+ medusa_s3_credentials: |-
+   [default]
+   aws_access_key_id = my_access_key
+   aws_secret_access_key = my_secret_key
+```
```
[the remark this answers is not in the document]
Comment (PrasadG193) [reply examples/k8ssandra/README.md:47]: @ihcsim the secret key has multi line value. I did not find any straight forward way. We can use `kubectl create sacret --from-file` but the file still has to be created and updated

File: examples/k8ssandra/k8ssandra-blueprint.yaml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        operation: create
+        namespace: '{{ .Object.metadata.namespace }}'
+        spec: |-
+          apiVersion: cassandra.k8ssandra.io/v1alpha1
+          kind: CassandraBackup
+          metadata:
+            name: kanister-{{ .Object.spec.clusterName }}-{{ .Object.metadata.name }}-{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02t15-04-05" }}
+          spec:
+            name: kanister-{{ .Object.spec.clusterName }}-{{ .Object.metadata.name }}-{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02t15-04-05" }}
+            cassandraDatacenter: "{{ .Object.metadata.name }}"
```
[the remark this answers is not in the document]
Comment (PrasadG193) [reply examples/k8ssandra/k8ssandra-blueprint.yaml:29]: @ihcsim we pass Datacenter resource as a subject while creating ActionSet. So we can refer to it with `.Object`.

[the remark this answers is not in the document]
Comment (ihcsim) [reply examples/k8ssandra/k8ssandra-blueprint.yaml:29]: Do we normally include sample actionset in examples?

**Review by ihcsim — APPROVED** [review]
LGTM - thanks for working on this!

File: examples/k8ssandra/README.md [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+  enabled: true
+  storageSecret: medusa-bucket-key
+  storage: s3
+```
+
+Pass the values.yaml file to `helm install` command to overwrite the default Helm chart configuration
+
+```bash
+# Create namespace
+$ kubectl create namespace k8ssandra
```
Comment (ihcsim) [inline examples/k8ssandra/README.md:96]: Can we remove this, now that we added `--create-namespace`?

File: examples/k8ssandra/README.md [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+
```
Comment (PrasadG193) [inline examples/k8ssandra/README.md:97]:
```suggestion
```

**Pull request merged**